text/0000-io-safety.md

```diff
@@ -136,9 +136,9 @@ be gradual.
 
 These two types are conceptual replacements for `RawFd`, and represent owned
 and borrowed handle values. `OwnedFd` owns a file descriptor, including closing
-it when it's dropped. `BorrowedFd`'s lifetime parameter ties it to the lifetime
-of something that owns a file descriptor. These types enforce all of their I/O
-safety invariants automatically.
+it when it's dropped. `BorrowedFd`'s lifetime parameter says for how long
+access to this file descriptor has been borrowed. These types enforce all of
+their I/O safety invariants automatically.
```

Comment:
Thanks!

With this change, I think I am wondering why the lifetime parameter is called `'owned`?

Reply:
I read `'owned` as saying "assume there's an `OwnedFd` (or equivalent) that this `BorrowedFd` can't outlive". I'm not attached to it, and recognize that that may be specific to how I think about lifetimes. I'd be ok with `'a` or another name here.

Reply:
Simply `'file` or `'fd` might be fine, because BorrowedFd is trying to represent the "inherent" lifetime of the file descriptor.

Reply:
Renamed to `'fd` (and `'handle' etc. on Windows).